### PR TITLE
metadata-hook: support group members other than host creating/editing shared resources

### PR DESCRIPTION
**Pull request by loganallenc**
Pretty small change, but nice quality of life improvement.

NOTE: this is good to keep in mind for links and publish, because the preference is now to send all metadata actions through %metadata-hook so that we have automatic support for foreign chatrooms / publish blogs / links collections being shown in the group's available channels. (not just the host's channels)

### CODE REVIEW

**Comment by loganallenc** [comment]
Note: I have one change incoming that makes it so that "leaving a chat that you are not the host of" doesn't affect the metadata of that chat.

File: pkg/arvo/mar/metadata/action.hoon [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/-  *metadata-store
+|_  act=metadata-action
+++  grab
```
Comment (Fang-) [inline pkg/arvo/mar/metadata/action.hoon]: Maybe also write the trivial `+noun:grow`, for completeness' sake.

Comment (loganallenc) [reply pkg/arvo/mar/metadata/action.hoon]: Matilde’s PR includes a grow arm to noun and json already. I’ll merge this when I get back from workout at noon

Comment (loganallenc) [reply pkg/arvo/mar/metadata/action.hoon]: Alternatively, feel free to merge sooner. The conflict in that file is trivial

**Pull request merged**